charts/kong/ci/__snapshots__/admin-api-service-clusterip-values.snap

```diff
@@ -235,11 +237,11 @@ spec:
           readinessProbe:
             failureThreshold: 3
             httpGet:
-              path: /status/ready
+              path: /status
```

Comment:
`/status/ready` was introduced since issue https://github.com/Kong/charts/issues/830

why change it to `/status`?

Reply:
https://github.com/Kong/charts/pull/1092#discussion_r1642941245